### ISSUE TITLE
Requesting new app: syft

### ISSUE THREAD

**Issue by arthurzenika** [wimpysworld/deb-get#1010]
## Requesting a new app ✨

If you are requesting a new app to be added to `deb-get` please complete the
following section, only one app per issue please.

 - [x] I have checked that this app does not already have a pending request and has not previously been declined.
 - App Name: syft
 - Download URL: https://github.com/anchore/syft/releases/download/v0.105.0/syft_0.105.0_linux_amd64.deb
 - Website URL: https://github.com/anchore/syft
 - App Summary:  CLI tool and library for generating a Software Bill of Materials from container images and filesystems

**Comment by philclifford** [comment]
Already supported

```
$ deb-get show syft
Syft
  Package:	syft
  Repository:	01-main
  Updater:	deb-get
  Installed:	0.105.0
  Published:	0.105.0
  Architecture:	amd64 arm64
  Download:	https://github.com/anchore/syft/releases/download/v0.105.0/syft_0.105.0_linux_amd64.deb
  Website:	https://github.com/anchore/syft
  Summary:	CLI tool and library for generating a Software Bill of Materials from container images and filesystems.
```

**Comment by arthurzenika** [comment]
Sorry.
